packages/loopring_v3/contracts/test/AmmPool.sol

```diff
@@ -204,17 +211,18 @@ contract AmmPool is IBlockReceiver, IAgent {
         }
     }
 
+    /// @param amounts The amounts to deposit + the amount of pool tokens to deposit
```

Comment:
Not sure I understand: why do we support depositing pool tokens (liquidity tokens)?